include/game/hsfformat.h

```diff
@@ -339,9 +341,23 @@ typedef struct hsf_bitmap_keyframe {
 typedef struct hsf_track {
     u8 type;
     u8 start;
-    u16 target;
-    u16 param;
-    u16 channel;
+    union {
+        u16 target;
+        s16 target_s16;
+    };
+    union {
+        s32 unk04;
+        struct {
+            union {
+                s16 param;
+                u16 param_u16;
+            };
+            union {
+                u16 channel;
+                s16 channel_s16;
+            };
+        };
+    };
```

Comment:
This one got pretty messy. I wonder if `HsfTrack` is actually a union of structs?